cmd/boulder-ra/main.go

```diff
@@ -91,6 +93,21 @@ type Config struct {
 		// you need to request a new challenge.
 		PendingAuthorizationLifetimeDays int `validate:"required,min=1,max=29"`
 
+		// ValidationProfiles is a map of validation profiles to their
+		// respective issuance allow lists.
+		ValidationProfiles map[string]struct {
+			// AllowList specifies the file path to a YAML list of account IDs
+			// permitted to use this profile. This field has three possible
+			// states:
+			//  1. Not included in the config file: No one is allowed to use
+			//     this profile.
+			//  2. Included in the config file with an empty string: Everyone
+			//     is allowed.
+			//  3. Included in the config file with a file path: Only the
+			//     account IDs listed in the file are allowed.
+			AllowList *string `validate:"omitempty"`
```

Comment:
I don't think this needs to be a pointer to a string, it can just be a string -- the string zero value is sufficient to communicate "no file has been configured for us to try to load".